src/createClassProxy.js

```diff
@@ -148,7 +148,9 @@ function proxyClass(InitialComponent) {
         const savedDescriptor = savedDescriptors[key];
 
         if (!isEqualDescriptor(prevDescriptor, savedDescriptor)) {
-          Object.defineProperty(NextComponent, key, prevDescriptor);
+          try {
+            Object.defineProperty(NextComponent, key, prevDescriptor);
+          } catch (err) { }
```

Comment:
I'd just add the comment into `catch` block what are the reasons for `try` block to fail.